### PR TITLE
Docs: Run multiple instances for new users

### PR DESCRIPTION
**Pull request by muletman**
This is my proposition of contribution for how new users could get up and running with multiple instances of the bot, based on the conversation I had on Slack with @hroff-1902 
It appeared to me that the transparent creation and usage of the sqlite databases, and the necessity to create other databases to run multiple bots at the same time was not so straightforward to me in the first place, despite browsing through the docs. It is evident now ;) but that will maybe save time for devs if any other new user come on slack with the same issue.

Thanks

Thank you for sending your pull request. But first, have you included
unit tests, and is your code PEP8 conformant? [More details](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md)

## Summary
Explain in one sentence the goal of this PR

Solve the issue: #___

## Quick changelog

- <change log #1>
- <change log #2>

## What's new?
*Explain in details what this PR solve or improve. You can include visuals.*

### CODE REVIEW

File: HowToRunMultipleInstances.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+As for various other things, upon docker or manual install, [freqtrade will create by default two different databases, one for dry-run, and the other for live trades.](https://www.freqtrade.io/en/latest/docker/#create-your-database-file)
+
+By default, executing the trade command in the command line interface, without specifying any database (`--db-url`) argument, freqtrade will store your trades and performance history in one of these two default databases, depending if dry-run mode is enabled or not. These databases are actual .sqlite files which are stored in your main freqtrade folder, under the name `tradesv3.dryrun.sqlite` for the dry-run mode, and `tradesv3.sqlite` for the live mode.
+
+The optional argument to the trade command used to specify the path of these files is `--db-url`. So when you are starting a bot with only the config and strategy arguments in dry-run mode for instance :
```
Comment (xmatthias) [inline HowToRunMultipleInstances.md:11]:
```suggestion
The optional argument to the trade command used to specify the path of these files is `--db-url`. So when you are starting a bot with only the config and strategy arguments in dry-run mode for instance:
```

File: HowToRunMultipleInstances.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+is equivalent to :
+
+```
+freqtrade trade -c MyConfig.json -s MyStrategy --db-url sqlite:///tradesv3.dryrun.sqlite
+```
+
+That means that if you are running the trade command in two different terminals, for example to test your strategy both for trades in USDT and in another instance for trades in BTC, you will have to run them with different databases. Even if you are using the same configuration file and strategy.
+
+ If you specify the URL of a database which does not exist, freqtrade will create one with the name you specified. So for example if you want to test your custom strategy in BTC vs USDT, you could type in one terminal :
```
Comment (xmatthias) [inline HowToRunMultipleInstances.md:25]:
```suggestion
 If you specify the URL of a database which does not exist, freqtrade will create one with the name you specified. So for example if you want to test your custom strategy in BTC vs USDT, you could type in one terminal:
```

File: HowToRunMultipleInstances.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+By default, executing the trade command in the command line interface, without specifying any database (`--db-url`) argument, freqtrade will store your trades and performance history in one of these two default databases, depending if dry-run mode is enabled or not. These databases are actual .sqlite files which are stored in your main freqtrade folder, under the name `tradesv3.dryrun.sqlite` for the dry-run mode, and `tradesv3.sqlite` for the live mode.
+
+The optional argument to the trade command used to specify the path of these files is `--db-url`. So when you are starting a bot with only the config and strategy arguments in dry-run mode for instance :
+
+```
+freqtrade trade -c MyConfig.json -s MyStrategy
+```
+
+is equivalent to :
```
Comment (xmatthias) [inline HowToRunMultipleInstances.md:17]:
```suggestion
is equivalent to:
```

File: HowToRunMultipleInstances.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+```
```
Comment (xmatthias) [inline HowToRunMultipleInstances.md:19]:
```suggestion
``` bash
```

File: HowToRunMultipleInstances.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+and in the other
+
+```
+freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite://path/path/tradesUSDT.live.sqlite
+```
+
+For more information regarding usage of the sqlite databases, for example to manually enter or remove trades, please refer to the following page :
+
+https://www.freqtrade.io/en/latest/sql_cheatsheet/
```
Comment (xmatthias) [inline HowToRunMultipleInstances.md:51]: maybe 

```suggestion
For more information regarding usage of the sqlite databases, for example to manually enter or remove trades, please refer to the [SQL Cheatsheet](sql_cheatsheet.md).
```

File: HowToRunMultipleInstances.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+```
+freqtrade trade -c MyConfig.json -s MyStrategy --db-url sqlite:///tradesv3.dryrun.sqlite
+```
+
+That means that if you are running the trade command in two different terminals, for example to test your strategy both for trades in USDT and in another instance for trades in BTC, you will have to run them with different databases. Even if you are using the same configuration file and strategy.
+
+ If you specify the URL of a database which does not exist, freqtrade will create one with the name you specified. So for example if you want to test your custom strategy in BTC vs USDT, you could type in one terminal :
+
+```
+freqtrade trade -c MyConfigBTC.json -s MyCustomStrategy --db-url sqlite://path/tradesBTC.dryrun.sqlite
```
Comment (xmatthias) [inline HowToRunMultipleInstances.md:28]: an sqlalchemy sqlite path resolving to a file should have 3 slashes for relative directories - and 4 slashes for absolute direcories.

So for example: `sqlite:///file_in_this_folder.sqlite` - or `sqlite:////home/user/absolute_path.sqlite` ... 

[source](https://docs.sqlalchemy.org/en/13/core/engines.html#sqlite)

**Comment by muletman** [comment]
Sorry guys, just saw your comments, 
still not used to push/pull requests... (but that is part of the learning process as well...)
On about creating new page for that I totally agree, maybe not necessary, it was just the simplest I guess given my lack of experience;
I'll try to follow that more closely next time.

See you on slack

**Comment by xmatthias** [comment]
No problem at all, noone demands you respond within time X ... :) 

also, receiving comments on a PR is something that's usual, overall, i think you did a great job :)

**Comment by muletman** [comment]
Thank you very much !

**Pull request merged**